docs/examples/use_cases/pytorch/efficientnet/image_classification/dataloaders.py

```diff
@@ -419,6 +436,161 @@ def get_pytorch_val_loader(
 
     return PrefetchedWrapper(val_loader, 0, num_classes, one_hot), len(val_loader)
 
+def read_file(path):
+    return np.fromfile(path, dtype=np.uint8)
+
+def read_filepath(path):
+    return np.frombuffer(path.encode(), dtype=np.int8)
+
+def get_dali_proxy_train_loader(dali_device='gpu', send_filepaths=False):
+    def get_impl(data_path,
+                 image_size,
+                 batch_size,
+                 num_classes,
+                 one_hot,
+                 interpolation="bilinear",
+                 augmentation=None,
+                 start_epoch=0,
+                 workers=5,
+                 _worker_init_fn=None,
+                 prefetch_factor=2,
+                 memory_format=torch.contiguous_format):
+        traindir = os.path.join(data_path, "train")
+
+        interpolation = {
+            "bicubic": types.INTERP_CUBIC,
+            "bilinear": types.INTERP_LINEAR,
+            "triangular": types.INTERP_TRIANGULAR,
+        }[interpolation]
+
+        output_layout = 'CHW' #"HWC" if memory_format == torch.channels_last else "CHW"
```

Comment:
This looks like something that should be uncommented

Reply:
will do